packages/mdc-animation/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.js
+!*.ts
```

Comment:
I'm confused, why does this even need to be here? Shouldn't we just need to ignore *.d.ts and be done?

For that matter, why does this `gitignore` include things only present under `dist`? The root `gitignore` already ignores `packages/*/dist`, which makes me wonder if this `gitignore` is completely overriding that, in which case to Andy's point we should definitely be working with the existing root `gitignore` instead.

Reply:
This is now
```
packages/**/*.d.ts
```